src/components/ui/SearchCategory.tsx

```
@@ -0,0 +1,93 @@
+  { id: "cameras", name: "Cameras", icon: "📷" },
+];
+
+const SearchIcon = () => (
+  <svg
+    className="h-5 w-5 text-gray-400"
+    fill="none"
+    stroke="currentColor"
+    viewBox="0 0 24 24"
+  >
+    <path
+      strokeLinecap="round"
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)